cpu-hog/prow_run.sh

```diff
@@ -15,19 +15,14 @@ SCENARIO_FOLDER="$krkn_loc/scenarios/arcaflow/cpu-hog"
 echo "Printing cluster version"
 oc version
 
-# Copy config to kraken
-cp node-cpu-hog/input.yaml.template $SCENARIO_FOLDER/input.yaml.template
-
 source node-cpu-hog/env.sh
-source env.sh
 source common_run.sh
 
 setup_arcaflow_env "$SCENARIO_FOLDER"
 checks
 
 # Substitute config with environment vars defined
-#envsubst < node-cpu-hog/input.yaml.template> node-cpu-hog/cpu_hog_scenario.yaml
```

Comment:
Why don't we need this scenario file anymore?

Reply:
Is it already set in the set arca scenario above?

Reply:
oh my bad, setup_arcaflow_env does the envsubst.